docs/running-jobs/job-resources.md

```diff
@@ -8,62 +8,27 @@ Job scripts, the `sbatch` command, and the `sinteractive` command support many d
-| ---------------- | ----------------- | --------- | --------- | ---------------- | ------------------------ |
-| atesting <sup>3</sup> | Testing      | Up to 2   | 16        | 0.25             | 0.5H, 3H                 |
-| acompile         | Compile           | 1         | 4         | 1.0              | 1H, 12H                  |
-| ainteractive     | Interactive Jobs  | 1         | 1         | 1.0              | 1H, 12H                  |
-
-> <sup>3</sup> The `atesting` partition is limited to 16 cores total. These cores can come from up to 2 nodes, but a user is limited to maximum of 16 cores per job.
-
-To run a job longer than 24 hours on the `amilan`, `ami100`, or `aa100` partitions, use the `long` QOS.
-
-More details about each type of node can be found [here](../clusters/alpine/alpine-hardware.md).
+Nodes with the same hardware configuration are grouped into partitions. You will need to specify a partition using `--partition` in your job script in order for your job to run on the appropriate type of node. A list of partitions available on Alpine can be found in our page on [Alpine Hardware](../clusters/alpine/alpine-hardware.md#partitions). 
 
 ## Quality of Service
 
-Quality of Service (QoS) is used to constrain or modify the characteristics that a job can have. This could come in the form of specifying a QoS to request for a longer run time or a high priority queue for condo owned nodes. For example, by selecting the `long` QoS, a user can place the job in a lower priority queue with a max wall time increased from 24 hours to 7 days. 
-
-**Normally, this slurm directive does not need to be set for most jobs. Only set a QoS when requesting a long or condo job.**
-
-The available QoS's for Alpine are:
-
-| QOS name    | Description                | Max walltime    | Max jobs/user | Node limits        | Partition limits | Priority Adjustment  |
-| ----------- | -------------------------- | --------------- | ------------- | ------------------ | ---------------- | ---------------------|
-| normal      | Default                    | 1D              | tbd           | tbd                | n/a                 | 0                    |
-| long        | Longer wall times          | 7D              | tbd           | tbd                | amilan,aa100,ami100 | 0                    |
-| mem         | High-memory jobs           | 7D              | tbd           | 12                 | amem only           | 0                    |
+Quality of Service (QoS) is used to constrain or modify the characteristics that a job can have. This could come in the form of specifying a QoS to request for a longer run time or a high priority queue for condo owned nodes. For example, by selecting the `long` QoS, a user can place the job in a lower priority queue with a max wall time increased from 24 hours to 7 days. A list of QOS codes available on Alpine can be found in our page on [Alpine Hardware](../clusters/alpine/alpine-hardware.md#quality-of-service-qos). 
```

Comment:
```suggestion
Quality of Service (QoS) is used to constrain or modify the characteristics that a job can have. This could come in the form of specifying a QoS to request for a longer run time or a high priority queue for condo owned nodes. For example, by selecting the `long` QoS, a user can place the job in a lower priority queue with a max wall time increased from 24 hours to 7 days. A list of QoS codes available on Alpine can be found on our [Alpine Hardware](../clusters/alpine/alpine-hardware.md#quality-of-service-qos) page. 
```